proton-j/src/main/java/org/apache/qpid/proton/message/impl/MessageImpl.java

```diff
@@ -514,7 +536,32 @@ public ApplicationProperties getApplicationProperties()
     @Override
     public Section getBody()
     {
-        return _body;
+        Section section = _body;
```

Comment:
For clarity I might be inclined to null check and return this, like the next method does.